utils/growpart/Makefile

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (C) 2022 Eero Vuojolahti
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=growpart
+PKG_SOURCE_DATE:=2022-02-04
+PKG_SOURCE_VERSION:=4cc4950394b3db0243cb0211d6a394d629129a2a
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_SOURCE_DATE).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/canonical/cloud-utils/tar.gz/$(PKG_SOURCE_VERSION)?
+PKG_HASH:=291b045d550d07b7271ce61d5912962b031df53dfa27edce6821b9222b880b3b
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_SOURCE_VERSION)
```

Comment:
do you really need this line?

Reply:
Don't know, I copied the Makefile from some other package and used it as a template.